docs/nginx.md

```diff
@@ -0,0 +1,42 @@
+# Nginx Receiver
+
+The `nginx` receiver can fetch stats from a NGINX instance using the mod_status endpoint.
+
+
+## Prerequisites
+
+[ngx_http_stub_status_module](http://nginx.org/en/docs/http/ngx_http_stub_status_module.html) must be enabled on the NGINX instance. This may require editing the NGINX configuration.
+
+
+## Configuration
+
+| Field                 | Default                   | Description |
+| ---                   | ---                       | ---         |
+| `type`                | required                  | Must be `nginx`. |
+| `endpoint`            | `http://localhost/status` | The url exposed by the NGINX stats module. |
+| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    mynginxmetrics:
+      type: nginx
+      endpoint: http://localhost:80/status
+      collection_interval: 30s
+  service:
+    pipelines:
+      nginxpipeline:
+        receivers:
+          - nginxmetrics
```

Comment:
The receiver ID (`nginxmetrics`) does not seem to match the ID (`mynginxmetrics`) defined above. This would likely throw a validation error.

Reply:
Fixed